boilerplates/react-union-boilerplate-basic/src/apps/SampleApp/index.ssr.js

```diff
@@ -0,0 +1,10 @@
+// NOTE: If you don't need SSR, remove this file and the `react-union-ssr-server` dependency.
```

Comment:
It would be better to create separate boilerplate with SSR support.

Reply:
Sure, I can do that  – I've been editing the basic boilerplate to prevent myself from changing the structure too much.

I've made some miscellaneous changes to this boilerplate, so I'll do them in the other boilerplates too.